docker/qchem.dockerfile

```diff
@@ -25,7 +25,8 @@ WORKDIR /opt/pennylane/qchem
 RUN git submodule update --init --recursive
 RUN pip install wheel && pip install openfermionpyscf && pip install -r requirements.txt \
     && python3 setup.py install \
-    && make test
+    && pip install -i https://test.pypi.org/simple/ pennylane-lightning --pre --upgrade \
+    && make test && make coverage
```

Comment:
Same here.  Will we need to revert this for each release?

Reply:
As above.